### ISSUE TITLE
Running the model....

### ISSUE THREAD

**Issue by Auth0rM0rgan** [Junshk/CinCGAN-pytorch#4]

**Comment by Auth0rM0rgan** [comment]
Hey @Junshk 

Finally, I have executed your code but I have some questions:

> => use gpu id: '0'
> Random Seed:  1193
> ===> Loading datasets
> ===> Building model
> ===> Setting GPU
> => no checkpoint found at 'model_total_4.pth'
> ===> Training Step 1.
> Epoch = 0, lr = 0.0002
> ===> Epoch[0](0/1000): Loss: idt 0.000000 2.370729 cyc 0.000000  1.735660 D 0.000000 0.973776, G: 0.000000 0.870190, psnr_hr: 5.089412, psnr_lr 5.857366 

1) Why some of the loss functions are stay 0.0000 ?
2) I downloaded the EDSR pre-train model but as you can see its written "no checkpoint found at 'model_total_4.pth' " is it okay?
